README.md

```diff
@@ -1,48 +1,113 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-> A fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
+<!-- PROJECT DESCRIPTION -->
 
-## Built With
+# 📖 Blog App <a name="about-project"></a>
 
-> Ruby
-> Ruby on Rails
-> HTML
-> CSS
+> The Blog App, is a website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-## Getting Started
+## 🛠 Built With <a name="built-with"></a>
 
-Clone this repo: https://github.com/DiegoMero/blog-app.git
+### Tech Stack <a name="tech-stack"></a>
 
-- Go to the repo directory from the comand line.
-- Run 
-```
-ruby main.rb
-```
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-## Author
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Login**
+- **Create Posts**
+- **Create comments and like posts**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone this repo: git@github.com:DiegoMero/blog-app.git
+
+- Open irb on comand line.
+- Run the code 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
 
 👤 **Diego Mero**
 - GitHub: [@DiegoMero](https://github.com/DiegoMero)
 - Twitter: [@Dimero18](https://twitter.com/Dimero18)
 - LinkedIn: [Diego](https://www.linkedin.com/in/diego-mero/)
 
+👤 **Josué Benavides**
+
+- GitHub: [@github](https://github.com/jdbs9514)
+- Twitter: [@twitter](https://twitter.com/JODA1015)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/macoin)
 
-## 🤝 Contributing
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Leave a comment if you like the app!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Thanks to Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Give a ⭐️ if you like this project!
+<!-- FAQ (optional) -->
 
-## Acknowledgments
+<!-- LICENSE -->
 
-- Thanks to my learning partners for their support and advices
+## 📝 License <a name="license"></a>
 
-## 📝 License
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [**OPTIONAL**] RECOMMENDED: I think it would be better if you fix the  broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or an other method change this `[MIT](./LICENSE)` to  `[MIT](./MIT.md)` then the created new file in your directory name it `MIT.md` and get its content **[here](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**